XamlControlsGallery/ControlPages/MenuBarPage.xaml

```diff
@@ -14,7 +14,7 @@
                               XamlSource="MenuBar\MenuBarSample1.txt">
             <StackPanel>
                 <TextBlock x:Name="SelectedOptionText" Text="" />
-                <muxc:MenuBar>
+                <muxc:MenuBar x:Name="Example1">
```

Comment:
`x:Name` allows UIA to find that element? I thought that's what AutomationProperties is about, not `x:Name`.

Reply:
It is inferred from the `x:Name` if provided in this manner. If you also provide the AutomationProperties value, that overrides the `x:Name` value.

Reply:
I did not know that, thanks for the explanation!